packages/plus/src/components/Conversation/__stories__/Message.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Conversation } from '..'
+import { Template } from './Template.stories'
+
+export const Message = Template.bind({})
+
+Message.args = {
+  children: [
+    <Conversation.Message avatar={<div>Message</div>}>
```

Comment:
In the `avatar` you can try to use `Avatar` component from `@ultraviolet/ui` I think it will give a better render of the story